libs/payments/ui/src/lib/client/components/CheckoutCheckbox.tsx

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+'use client';
+
+import * as HoverCard from '@radix-ui/react-hover-card';
+import { useState } from 'react';
+
+interface CheckoutCheckboxProps {
+  isRequired: boolean;
+  termsOfService: string;
+  privacyNotice: string;
+  notifyCheckboxChange: (isChecked: boolean) => void;
+}
+
+export function CheckoutCheckbox({
+  isRequired,
+  termsOfService,
+  privacyNotice,
+  notifyCheckboxChange,
+}: CheckoutCheckboxProps) {
+  const [isChecked, setIsChecked] = useState(false);
+
+  const changeHandler = () => {
+    const newValue = !isChecked;
+    setIsChecked(newValue);
+    notifyCheckboxChange(newValue);
+  };
+
+  return (
+    <div className="my-6">
+      <HoverCard.Root open={isRequired && !isChecked}>
+        <label className="flex gap-5 items-center">
+          <HoverCard.Trigger>
+            <input
+              type="checkbox"
+              name="confirm"
+              className="grow-0 shrink-0 basis-4 scale-150 cursor-pointer"
+              checked={isChecked}
+              onChange={changeHandler}
+            />
+          </HoverCard.Trigger>
+          <span className="font-normal text-[13px] block">
```

Comment:
While tabbing, it goes from the checkbox to the "Test handle error" button - I am unable to tab over to either "Terms of Service" or "Privacy Notice".

Reply:
This is the correct behavior. Links shouldn't be tabbed to.